doc/s3ip_sysfs/s3ip_sysfs_specification.md

```diff
@@ -0,0 +1,239 @@
+|/sys_switch/psu/psu[n]/hardware_version |RO|  string  | PSU hardware version number
+|/sys_switch/psu/psu[n]/serial_number |RO|  string  | PSU serial number
+|/sys_switch/psu/psu[n]/part_number |RO|  string  | PSU part number
+|/sys_switch/psu/psu[n]/type|RO| enum | PSU Type: <br>0: DC<br> 1: AC 
+|/sys_switch/psu/psu[n]/in_curr |RO|  int | PSU input current, unit: mA
+|/sys_switch/psu/psu[n]/in_vol |RO|  int | PSU input voltage, unit: mV
+|/sys_switch/psu/psu[n]/in_power |RO|  int | PSU input power, compute it from in_curr * in_vol, unit: uW
+|/sys_switch/psu/psu[n]/out_max_power |RO|  int |The maximum output power of the PSU, unit: uW
+|/sys_switch/psu/psu[n]/out_curr |RO|  int | PSU output current, unit: mA
+|/sys_switch/psu/psu[n]/out_vol |RO|  int | PSU output voltage, unit: mV
+|/sys_switch/psu/psu[n]/out_power |RO|  int | PSU output power, compute it from out_curr * out_vol, unit: uW
+|/sys_switch/psu/psu[n]/num_temp_sensors |RO|  int | number of temperature sensors
+|/sys_switch/psu/psu[n]/temp[n]  | R/W| | reference temperature sensor definition
+|/sys_switch/psu/psu[n]/num_power_sensors |RO|  int | number of power sensors, TBD
+|/sys_switch/psu/psu[n]/power_sensor[n]  | R/W| | reference power sensor definition, TBD
+|/sys_switch/psu/psu[n]/present|RO| enum | state:<br> 0: not present<br>1: Incumbent
+|/sys_switch/psu/psu[n]/out_status|RO| enum | Output status, via POWER_OK inside the power supply pin judgment<br>0: abnormal<br>1: normal
+|/sys_switch/psu/psu[n]/in_status|RO| enum | Input status, judged by AC_OK pin inside the power supply broken<br>0: abnormal<br>1: normal
+|/sys_switch/psu/psu[n]/fan_speed  |RO|  int | PSU fan speed, unit: RPM
+|/sys_switch/psu/psu[n]/fan_ratio | R/W|  Int| PSU fan speed duty cycle
+|/sys_switch/psu/psu[n]/led_status|RO|  enum | The PSU status lights are defined as follows:<br>See the definition of enumeration value of LED status light for details
+
+### 8. Transceiver information sysfs
+Transceiver module information Sysfs path must be /sys_switch/transceiver/
+
+*Table8-1 Transceiver module information sysfs Property*
+
+|Sysfs path|Permissions|Data type|Description|
+|-|-|-|-|
+|/sys_switchtransceiver/power_on |R/W| enum |Whether the port of the whole machine is powered on:<br>0: not powered on<br>1: Powered on
+|/sys_switchtransceiver/eth[n]/power_on  |R/W| enum |Whether the module is powered on:<br>0: not powered on<br>1: Powered on
+|/sys_switchtransceiver/eth[n]/tx_fault|RO| enum |module sending channel exception (including laser/TXCDR)<br>0: normal<br>1: abnormal.<br>This information is exported by eeprom (eg QSFP28 standard), the node may not support;Otherwise, the node must support (such as SFP28 standard)
+|/sys_switchtransceiver/eth[n]/tx_disable|R/W| enum |Development light<br>0: develop light<br>1: Turn off the light.<br>This information is exported by eeprom (eg QSFP28 standard), the node may not support;Otherwise, the node must support (such as SFP28 standard)

```

Comment:
What is the purpose of "Development Light?"

Reply:
It was a translation error and has been revised